process/subgroups.md

```diff
@@ -0,0 +1,66 @@
+# Subgroups
+
+Subgroups of the CG are created when one (or both) of the following apply:
+
+- An area of interest that needs a lot of discussion. Creating a subgroup with regular meetings provides more bandwidth for this discussion.
+- An area of interest is only relevant to a small group of participants. In these cases, subgroup meetings give a more focused venue. This way, participants don’t have concerns about wasting the CG’s time with something that the rest of the CG is not engaged in.
+
+## Active Subgroups
+
+| Subgroup  | Chair(s)  | Discussion Repo  | Meetings  |
+|---|---|---|---|
+| GC  | Thomas Lively (@tlively) + others?  | [Repo](https://github.com/WebAssembly/gc)  | [Agendas](https://github.com/WebAssembly/gc/issues?q=is%3Aissue+%22Agenda+for+subgroup+meeting%22)  |
+| SIMD  | Petr Penzin (@penzn) + others?  | ? | [Agendas](https://github.com/WebAssembly/meetings/tree/main/simd/)  |
+| Stack  | Francis McCabe (@fgmccabe) + others?  | [Repo](https://github.com/WebAssembly/stack-switching)  | [Agendas](https://github.com/WebAssembly/meetings/tree/main/stack)  |
```

Comment:
@fgmccabe I think this name is used as a mirror to the directories in the meeting repo. Wouldyou prefer Stack Switching here instead of Stack?

Reply:
Stack Switching is more clear, so I went ahead and updated that.